src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -177,6 +179,12 @@ def _get_shell_type():
     else:
         return _remove_cmd_chars(_remove_symbols(os.environ.get('SHELL')))
 
+def _get_error_hash():
+    err = sys.exc_info()[1]
+    return _remove_cmd_chars(_remove_symbols(_get_hash(str(err.message))))
```

Comment:
I had it that way originally, but changed it for some reason.  I put it back to plain str(err)

---
In reply to: [80156762](https://github.com/Azure/azure-cli/pull/959#discussion_r80156762) [](ancestors = 80156762)